assets/js/i18n/locales.js

```diff
@@ -1,16 +1,14 @@
-import { t } from '@lingui/macro'
 
-export const supportedLocales = {
-  en: t('js.i18n.locales.en')`English`,
-  el: t('js.i18n.locales.el')`Greek`
-}
+export const supportedLocalesData = window && window.i18nConfig && window.i18nConfig.localesData ? window.i18nConfig.localesData : [
+  { locale_id: 'en', locale_name: 'English' }
+]
 
-export const supportedLocaleIds = Object.keys(supportedLocales)
+export const supportedLocaleIds = supportedLocalesData.map(localeData => localeData.locale_id)
 
 const defaultLocale = 'en'
```

Comment:
I know this is a bit beyond the scope of this pull request, but I have another favor to ask:

Could these variables please be called `defaultLocaleId` and `currentLocaleId`?

I was going to make the change myself, but I figured you're already in the code ... so if I edit, it could make merges more difficult.